components/client/src/test/unit/java/com/hotels/styx/client/loadbalancing/strategies/BusyConnectionsStrategyTest.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2013-2018 Expedia Inc.
- *
+ * <p>
```

Comment:
`<p>`s should not be here. Note that we now have an issue relating to this ongoing problem: https://github.com/HotelsDotCom/styx/issues/107